keps/sig-node/2033-kubelet-in-userns-aka-rootless/README.md

```diff
@@ -654,67 +818,211 @@ Note that this feature gate does not support separating kubelet's user namespace
+-->
+
+No
 
 ### Monitoring Requirements
 
-_This section must be completed when targeting beta graduation to a release._
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
 
-* **How can an operator determine if the feature is in use by workloads?**
+###### How can an operator determine if the feature is in use by workloads?
 
-N/A
+<!--
+Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
+checking if there are objects with field X set) may be a last resort. Avoid
+logs or events for this purpose.
+-->
 
-* **What are the SLIs (Service Level Indicators) an operator can use to determine 
-the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [X] Other (treat as last resort)
-    - Details: Use `systemctl --user is-system-running` to verify whether the processes (RootlessKit, kubelet, kube-proxy, and CRI) are running.
+They can determine if a Pod is running on a node that is running in UserNS.
 
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+###### How can someone using this feature know that it is working for their instance?
 
-N/A
+<!--
+For instance, if this is a pod-related feature, it should be possible to determine if the feature is functioning properly
+for each individual pod.
+Pick one more of these and delete the rest.
+Please describe all items visible to end users below with sufficient detail so that they can verify correct enablement
+and operation of this feature.
+Recall that end users cannot usually observe component logs or access metrics.
+-->
 
-* **Are there any missing metrics that would be useful to have to improve observability 
-of this feature?**
+- [X] Events
+  - Event Reason:  No CrashLoopBackOff
+- [ ] API .status
+  - Condition name: 
+  - Other field: 
+- [ ] Other (treat as last resort)
+  - Details:
 
-N/A, but it'd be useful to have the kubelet publish whether or not it is running rootless, as a boolean metric.
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+99.9% of /health requests per day finish with 200 code
```

Comment:
You may want to check https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md for ideas how to improve this and following section.

Reply:
Is there any actual KEP to follow as a good example?

Couldn't find actual KEPs following the text of ["Steady state SLIs/SLOs"](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#steady-state-slisslos), except this one:
https://github.com/kubernetes/enhancements/blob/2a2b7ca5fa2982b029609b17ea71f7bd0da327ee/keps/sig-node/4800-cpumanager-split-uncorecache/README.md?plain=1#L416-L419